lib/kitchen/provisioner/base.rb

```diff
@@ -182,6 +181,13 @@ def sandbox_path
           "trying to access the path."
       end
 
+      # Returns the list of items in the sandbox directory
+      #
+      # @return [String] path of items in the sandbox directory
+      def sandbox_dirs
+        Util.list_directory sandbox_path
```

Comment:
Bit picky, but might be good to adjust to the style on `lib/kitchen/verifier/base.rb` ?

Reply:
Updated. Thanks @tecracer-theinen